packages/markdown/src/components/MarkdownPreview.tsx

```diff
@@ -199,7 +202,7 @@ export const MarkdownPreview = React.memo((props: MarkdownPreviewProps) => {
   // See the list of allowed Tags here:
   // https://github.com/cure53/DOMPurify/blob/main/src/tags.js#L3-L121
   const cleanHTML = React.useMemo(() => {
-    return DOMPurify.sanitize(props.value);
+    return replaceMailtoAmp(DOMPurify.sanitize(props.value));
```

Comment:
might want to look at 
https://github.com/cure53/DOMPurify/blob/f5c3e0b3f8c17b4d3fa0a827038f0a91e2e244ad/README.md?plain=1#L280

Reply:
Unfortunately this did not fix the issue